driver-core/src/main/com/mongodb/internal/operation/ChangeStreamBatchCursor.java

```diff
@@ -161,17 +166,24 @@ private void cachePostBatchResumeToken(final AggregateResponseBatchCursor<RawBso
         }
     }
 
-    private List<T> convertResults(final List<RawBsonDocument> rawDocuments) {
+    /**
+     * @param lastIdConsumer Is {@linkplain Consumer#accept(Object) called} iff {@code rawDocuments} is successfully converted
```

Comment:
Nit: typo `iff`

Reply:
Actually, it's not a typo but a short for ["if and only if"](https://en.wikipedia.org/wiki/If_and_only_if). It'd be nice if we agreed to adapt this shortening.

Do you still want me to change it to "if and only if"?

Reply:
TIL :) I'm happy with if or iff in this case.